src/modules/programs/components/program-filterable-list/program-list-container/program-list/program-item/program-item.js

```diff
@@ -1,30 +1,28 @@
 import classnames from "classnames";
 import React from "react";
-
-import PIButtons from "../../../../../../../components/program-item/pi-buttons/pi-buttons";
+import moment from "moment";
+import PTIButtons from "../../../../../../../components/program-item/pi-buttons/pi-buttons";
 import PIChart from "../../../../../../../components/program-item/pi-chart/pi-chart";
 import PIInfo from "../../../../../../../components/program-item/pi-info/pi-info";
 import PIStatistic from "./pi-statistic/pi-statistic";
 
 import "./program-item.css";
 
 const ProgramItem = ({ program, isAuthenticated, openInvestPopup }) => {
-  const programChartData = program.chart.map(x => ({
-    fund: +(x.investorFund + x.managerFund).toFixed(8),
-    profit: x.profit,
-    loss: x.loss,
-    totalProfit: x.totalProfit
+  const programChartData = program.equityChart.map(x => ({
+    value: x.value,
+    name: moment(x.date).format("MMM Do")
```

Comment:
Points on the chart will be sorted by "MMM Do". For example April 1st wil place early than March 31st. Use  Date.getTime() instead. And use format function for axis ticks.